Tests/MoyaSugarTests/MoyaSugarProviderTests.swift

```diff
@@ -69,19 +69,6 @@ class MoyaSugarProviderTests: XCTestCase {
     let provider = MoyaSugarProvider<GitHubAPI>(endpointClosure: endpointClosure)
     let endpoint = provider.endpoint(GitHubAPI.userRepos(owner: "devxoul"))
     XCTAssertNotNil(endpoint.httpHeaderFields)
-    XCTAssertEqual(endpoint.httpHeaderFields!, ["Authorization": "MySecretToken"])
-  }
-
-  func testHTTPHeaderFields_extendedByEndpointClosure() {
```

Comment:
Why is this test deleted?

Reply:
This do the same as `testHTTPHeaderFields_overridedByEndpointClosure`, Because `MoyaProvider.defaultEndpointMapping(for: target)` carry on its own header(i.e. `["Accept": "application/json"]`).
It's duplicated.

Reply:
This do the same as `testHTTPHeaderFields_overridedByEndpointClosure`, Because `MoyaProvider.defaultEndpointMapping(for: target)` bring its own header on (i.e. `["Accept": "application/json"]`).

Reply:
Oh I see. Thanks.